pkgs/tools/security/nuclei/default.nix

```diff
@@ -36,6 +36,7 @@ buildGoModule rec {
     homepage = "https://github.com/projectdiscovery/nuclei";
     changelog = "https://github.com/projectdiscovery/nuclei/releases/tag/v${version}";
     license = licenses.mit;
-    maintainers = with maintainers; [ fab ];
+    maintainers = with maintainers; [ fab Misaka13514 ];
+    platforms = with platforms; unix ++ windows;
```

Comment:
`buildGoModule` [defaults](https://github.com/NixOS/nixpkgs/blob/1c9db9710cb23d60570ad4d7ab829c2d34403de3/pkgs/build-support/go/module.nix#L295) to `go.meta.platforms`, I'm not sure that this does anything

```suggestion
```